lib/rubocop/cop/rspec/align_left_let_brace.rb

```diff
@@ -1,49 +0,0 @@
-# frozen_string_literal: true
```

Comment:
The trick as I remember it was to wipe out files that won’t be present in the target state of the repo before the PR. Not remove them in the PR. Exceptions are lib/rubocop-rspec.rb -> lib/rubocop-rspec_rails.rb. At least this is how we did it for capybara https://github.com/rubocop-hq/rubocop-capybara/pull/1#issuecomment-1363270769

the difference is that in the final resulting repo we won’t keep much of the history of unrelated files (eg factory bot cops and docs).

Reply:
Oops, it seems that the deleted files were also included in the PR when extracting the FacotryBot division.
https://github.com/ydah/rubocop-factory_bot/pull/1

The steps at the time of extraction also remained below
https://github.com/rubocop/rubocop-factory_bot/pull/1#issuecomment-1507154570

Reply:
Ouch, that would undermine the whole approach. We could just have copied the whole repo and made necessary changes to turn it into the result. But the problem is that we keep unrelated history.
Do we care? I don’t know. We keep the history of files that remain anyway, like lib/rubocop-<gemname>.rb, changelog, docs.

Even though the PR in your repo does seem to include deleted files, the other one, in rubocop namespace, doesn’t with a mere 40 files touched in the PR. Also if you take a look at the [repo before the PR](https://github.com/rubocop/rubocop-factory_bot/tree/812771eb33453e40f51c7a1cc993d7c8b6f5b33a/lib/rubocop/cop), there are only factory_bot cops there.
I suggest to follow the same approach.

Reply:
I think they were probably taking care of the logs in some way. I'll try the previous procedure again this time and see if it's no problem.